packages/store/src/IStore.sol

```diff
@@ -7,6 +7,10 @@ import { Schema } from "./Schema.sol";
 import { IStoreHook } from "./IStoreHook.sol";
 
 interface IStoreRead {
+  event HelloStore(bytes32 indexed storeVersion);
+
+  function storeVersion() external view returns (bytes32);
```

Comment:
I really wanted this to just be `version()` but this overlaps with `World.version()` since World is a Store.